src/components/numbered-code-snippet/numbered-code-snippet.js

```diff
@@ -249,7 +254,6 @@ export default class NumberedCodeSnippet extends React.PureComponent {
     const highlightElements = [];
     const copyElements = [];
     let previousCount = 0;
-    let liveChunkCount = -1; // Incremented to give CopyButtons an identifier
```

Comment:
:bulb: Since all instances of NumberedCodeSnippet where using an empty arrow function of onCopy, the chunk identifier hasn't been in use. I also don't think we need it, I think we'd rather know that users are copying code and not necessarily which code block.

Reply:
I agree!